autopush/db.py

```diff
@@ -437,6 +442,9 @@ def delete_all_for_user(self, uaid):
         chidmessageids = [x["chidmessageid"] for x in results]
         if chidmessageids:
             self.delete_messages(uaid, chidmessageids)
+        self.table.delete_item(uaid=uaid,
```

Comment:
Would it work if you changed line 438 to `chidmessageid__gte=" "` and removed this?